### DISCUSSION TITLE
How to save prediction image

### DISCUSSION THREAD

**Discussion by mineeuk** [Project-MONAI/MONAI#5791] [Q&A] [answered]
I'm using UNETR code a brain image dataset instead of BTCV, 
but the prediction image is not saved. Is there any other way?

**Comment by KumoLiu** [comment, marked as answer]
Hi @ellenyumin, you can use `SaveImage` to save the predictions.
https://github.com/Project-MONAI/MONAI/blob/97918e46e0d2700c050e678d72e3edb35afbd737/monai/transforms/io/array.py#L292
```
saver = SaveImage(output_dir="./output", output_ext=".nii.gz", output_postfix="seg")
saver(output)
```
Thanks!
